harness/determined/cli/dev.py

```diff
@@ -309,8 +312,11 @@ def call_bindings(args: Namespace) -> None:
             + f"\n\n{str(e)}",
             e,
         )
-
-    print_response(output)
+    except (api_errors.BadRequestException, api_errors.BadResponseException) as e:
```

Comment:
good question. We throw both BadRequestException and ApiException via do_request method on session which is used in our bindings we want to catch those two and no really the BadRespnoseException. Since ApiException subclasses BadRequestException just catching the latter should be fine but let's catch both. They way these are organized and named are confusing to me and I suspect we might want to reorganize them later on. I'll update this